src/reducers/chat.js

```diff
@@ -23,6 +23,7 @@ export default function chat(state = { messages: [] }, action) {
     case CHAT_MESSAGE_IS_TYPING:
       return {
         ...state,
+        messages: Array.from(messages),
```

Comment:
You create a copy here? What for?

Reply:
The same trouble whith empty array

Reply:
We can have a call for you to explain, because I don't get it now.